common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -20,25 +21,49 @@ object SupplierProcessors {
     PaParser,
     ReutersParser,
     RexParser,
-    StaffPhotographerParser
+    PhotographerParser
   )
 
   def process(image: Image): Image =
     all.foldLeft(image) { case (im, processor) => processor(im) }
 }
 
-object StaffPhotographerParser extends ImageProcessor {
+object PhotographerParser extends ImageProcessor {
   def apply(image: Image): Image = {
-    image.metadata.byline.map(p => (p,StaffPhotographers.getPublication(p))) match {
-      case Some((photographer, Some(publication))) => image.copy(
-        usageRights = StaffPhotographer(photographer, publication),
-        metadata    = image.metadata.copy(credit = Some(publication))
-      )
-      case _ => image
+    image.metadata.byline.map { byline =>
+      (
+        byline,
+        image.metadata.credit.map(_.toLowerCase),
+        PhotographersList.getPublication(staffPhotographers, byline),
+        PhotographersList.getPublication(contractedPhotographers, byline)
+      ) match {
+        // staff photographer
+        case (byline, credit, Some(publication), _) => image.copy(
+          usageRights = StaffPhotographer(byline, publication),
+          metadata    = image.metadata.copy(credit = Some(publication))
+        )
+
+        // contracted photographer
+        case (byline, credit, None, Some(publication)) => image.copy(
+          usageRights = ContractPhotographer(byline, publication),
+          metadata    = image.metadata.copy(credit = Some(publication))
+        )
+
+        // commissioned photographer
+        case (byline, Some("the guardian"), None, None) => image.copy(
+          usageRights = CommissionedPhotographer(byline, "The Guardian")
+        )
+        case (byline, Some("the observer"), None, None) => image.copy(
+          usageRights = CommissionedPhotographer(byline, "The Observer")
+        )
```

Comment:
Some images that come through from the past are not on our staff / contract lists as we just don't
have all the data from RCS.

This will then account for those not on those lists, but we have explicitly set the credit to be ourselves.